p4runtime_sh/test.py

```diff
@@ -417,6 +417,36 @@ def test_table_entry_ternary_dont_care(self):
         with self.assertRaisesRegex(UserError, "ternary don't care match"):
             te.match["header_test.field32"] = "10.0.0.0&&&0.0.0.0"
 
+    def test_string_match_ekey(self):
+        te = sh.TableEntry("StringMatchKeyTable")(action="actionA")
+        te.match["f13"] = "16"
+        te.action["param"] = "aa:bb:cc:dd:ee:ff"
+        te.insert()
+
+        expected_entry = """
+table_id: 33554507
+match {
+  field_id: 1
+  exact {
+    value: "16"
+  }
+}
+action {
+  action {
+    action_id: 16783703
+    params {
+      param_id: 1
+      value: "\\x00\\x00\\x11\\x00\\x22\\x33"
```

Comment:
looks like this is wrong

Reply:
Yeah. Just noticed and fixed it. Thanks.